packages/viewer/src/lib/interactions/baseInteractionHandler.ts

```diff
@@ -152,6 +155,13 @@ export abstract class BaseInteractionHandler implements InteractionHandler {
       }
     }, this.getConfig().interactions.interactionDelay);
 
+    window.setTimeout(() => {
+      if (this.clickCount === 2) {
+        this.handleDoubleClick(event);
+      }
+      this.clickCount = 0;
+    }, this.getConfig().interactions.doubleClickDelay);
```

Comment:
Can we try with the native `dblclick` event before rolling something custom? https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event

Reply:
We can't use the native dblclick event as it is only for the primary mouse button (not the middle mouse button). However, we could use the mousedown event. Do you like this approach better?